client/app/pages/dashboards/hooks/useAddDashboardToRecentDashboards.js

```diff
@@ -0,0 +1,31 @@
+import { reduce, map, find } from "lodash";
+import { useCallback } from "react";
+
+export default function useAddDashboardToRecentDashboards(dashboardId) {
```

Comment:
I agree with @gabrieldutra that we should consolidate the code and have a single hook to add into recents that can work both for queries and dashboards (and actually other objects like alerts as well). Also the code when reading recents probably can be consolidated as well.

There is alot of duplicate between both transforms/hooks.

Reply:
I also agree with you guys, i will work on it.

Reply:
@arikfr All done!